### DISCUSSION TITLE
v1.7.1 - Extra Effects (Audio Units) hosting, Stability Improvements, ready for Lifetime License

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#768] [General]
#### Features & Improvements:
* Host 3rd party AU (Audio Units) in the eqMac audio pipeline
* Announcements/News section right within the app
* Volume remapping/limit
* Ready to receive and generate Lifetime Licenses once they go live
* Consistent Presets user experience
* Improved latency compensation calculations

#### Bug Fixes:
* Improved stability in every parts of the app
* Fixed Spatial Audio preset being reset
* Fixed a few Driver related crashes

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.7.1'>v1.7.1 - Extra Effects (Audio Units) hosting, Stability Improvements, ready for Lifetime License</a>.</em>

**Comment by Dragularos** [comment]
Doesn't work on OS 10.13.6. I've just updated from a working and stable earlier version. After zu update nothing happens any more. Restart doesn't work either. Where can download the old version somewhere here on GitHub? ... greetings and have a wonderful Christmas time!

**Comment by Wallen27** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
